instrumentation/spring-rabbit/README.md

```diff
@@ -0,0 +1,23 @@
+# brave-instrumentation-spring-rabbit
+
+## Tracing for Spring AMQP (RabbitMQ)
```

Comment:
Let's just remove the AMQP part and call it Spring Rabbit here and in the description